packages/app/src/app/pages/Pro/index.tsx

```diff
@@ -46,12 +43,7 @@ export const ProPage: React.FC = () => {
     return <ProLegacy />;
   }
 
-  // When  you are on pro?workspaceId=... the flow is set to manage existing workspace
-  // either by upgrading to pro or managing the subscription
-  if (urlWorkspaceId) {
-    return <ProUpgrade />;
-  }
+  return <ProUpgrade />;
 
-  // When you are on the /pro page, the flow is set to create a new pro subscription
-  return <ProCreate />;
+  // TODO: Cleanup ProCreate which is not used anymore
```

Comment:
Is this a valid TODO?

Reply:
Yes, I think it's safe to remove that component, but I would wait until we have the first test completed on stream.